VstsTfsSample/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("SimpleAdalConsoleApp")]
```

Comment:
The names here don't match the folder name.  I also don't love "VstsTfsSample" as the name.  What about "DualSupportSample"?